Sources/XMTPiOS/Conversations.swift

```diff
@@ -587,7 +587,9 @@ public actor Conversations {
 	}
 	
 	public func getHmacKeys(request: Xmtp_KeystoreApi_V1_GetConversationHmacKeysRequest? = nil) -> Xmtp_KeystoreApi_V1_GetConversationHmacKeysResponse {
-		let thirtyDayPeriodsSinceEpoch = Int(Date().timeIntervalSince1970) / (60 * 60 * 24 * 30)
+		let daysSinceEpoch = Date().timeIntervalSince1970 / (60 * 60 * 24)
+		let thirtyDayPeriodsSinceEpoch = Int(floor(daysSinceEpoch / 30))
```

Comment:
JS divides also by 1000 every where but I think that might be a bug in JS @rygine